components/sections/SponsorShowcase.tsx

```diff
@@ -55,56 +63,120 @@ function SponsorShowcase(): ReactElement {
           justifyItems="center"
           alignItems="center"
         >
-          <SponsorImage
-            src={`${imgPath}/deloitte.png`}
+          <SponsorLink
+            href="https://www2.deloitte.com/ca/en.html"
             gridArea="deloitte"
-            width={["50%", "50%", "70%"]}
-          />
-          <SponsorImage
-            src={`${imgPath}/innovapost.png`}
+            target="_blank"
+            rel="noopener noreferrer"
```

Comment:
Nitpick: you can use the `isExternal` prop as a shorthand for these (same with the rest of the images)

https://v0.chakra-ui.com/link#external-link